index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <header><h1>Hello Microverse</h1></header> 
+</body>
```

Comment:
- [ ]  I recommend using one class to styles for your header. Just Like this 
` <h1 class="heading">Hello `Microverse</h1>``